x-pack/plugins/aiops/public/components/explain_log_rate_spikes/explain_log_rate_spikes_analysis.tsx

```diff
@@ -284,13 +285,19 @@ export const ExplainLogRateSpikesAnalysis: FC<ExplainLogRateSpikesAnalysisProps>
           groupTableItems={groupTableItems}
           loading={isRunning}
           dataViewId={dataView.id}
+          dataView={dataView}
```

Comment:
If we need to pass in the full `dataView` let's try to get rid of the `dataViewId` prop and access the `id` from `dataView` within the component.

Reply:
Removed in a44cadef987c031223faa4069f24946f7d765920